### ISSUE TITLE
Buftabline in command bar?

### ISSUE THREAD

**Issue by tuyenpm9** [ap/vim-buftabline#47]
I like the way bling/vim-bufferline displays buffer list in command bar but it lacks mappings options, make it hard to switch between buffers, any thought about it?

**Comment by ap** [comment]
What options do you think should it have?

My thoughts are that it’s not this plugin that lacks convenient buffer switching, it’s Vim.

But it’s easy to make buffer switching more convenient in your `~/.vimrc`… and leaving the choice to the user lets them pick the way they like. The simplest way (which is essentially what I use myself) is shown in the README. I’m not putting that bit in the plugin because then I’d have to add a setting to turn it off… and it’s so trivial it just doesn’t add enough value to justify that cost.

What’s not easy is getting a display of the buffer list. Hence this plugin. That’s its value proposition.

Adding more comprehensive buffer management might have value… but I’m not really the right person for that, since I don’t use any. And there are *tons* of plugins that address buffer navigation in some way – I don’t want to duplicate that functionality, and I especially don’t want to conflict with them. So I’d rather keep buftabline easy to combine with whatever buffer manager plugin you like.

Plugins that come with a bunch of mappings demand an investment from the user. I’d like this plugin to be along the lines of <b>You drop it into your Vim, It Just Works, and then you go on to learn about managing buffers in Vim</b>… instead of learning buftabline.

Basically, I’d prefer it to be the missing buffer display built into Vim, as much as possible. (It kinda boggles the mind that Vim ships a built-in tab display but not a buffer display.)

Having said all that… I am open to considering suggestions contrary to that idea, even if I’m not likely to go that way. Particularly if you have an idea that would be hard to do as a separate plugin, I would be interested in hearing about it – though I’m trying hard to make sure such a thing cannot exist. 😊

**Comment by tuyenpm9** [comment]
Thanks for clarification, you did a great work with this plugin.
